test/summary/clearlydefinedTests.js

```diff
@@ -140,17 +146,55 @@ describe('ClearlyDescribedSummarizer addNpmData', () => {
 
     for (let license of Object.keys(data)) {
       let result = {}
-      summarizer.addNpmData(result, { registryData: { manifest: { license: license } } })
-      if (data[license]) assert.deepEqual(result, { licensed: { declared: data[license] } })
-      else assert.deepEqual(result, {})
+      summarizer.addNpmData(result, { registryData: { manifest: { license: license } } }, npmTestCoordinates)
+      if (data[license])
+        assert.deepEqual(result, {
+          described: {
+            urls: {
+              download: 'https://registry.npmjs.com/test/-/test-1.0.tgz',
+              registry: 'https://npmjs.com/package/test',
+              version: 'https://npmjs.com/package/test/v/1.0'
+            }
+          },
+          licensed: { declared: data[license] }
+        })
+      else
+        assert.deepEqual(result, {
+          described: {
+            urls: {
+              download: 'https://registry.npmjs.com/test/-/test-1.0.tgz',
+              registry: 'https://npmjs.com/package/test',
+              version: 'https://npmjs.com/package/test/v/1.0'
+            }
+          }
+        })
     }
 
     // should work in the type field as well
     for (let license of Object.keys(data)) {
       let result = {}
-      summarizer.addNpmData(result, { registryData: { manifest: { license: { type: license } } } })
-      if (data[license]) assert.deepEqual(result, { licensed: { declared: data[license] } })
-      else assert.deepEqual(result, {})
+      summarizer.addNpmData(result, { registryData: { manifest: { license: { type: license } } } }, npmTestCoordinates)
+      if (data[license])
+        assert.deepEqual(result, {
+          described: {
```

Comment:
this duplicate code can be collapsed by conditionally creating the target opbject and then asserting. So 

```js
const expected = {
  described: {
    urls: {
      download: 'https://registry.npmjs.com/test/-/test-1.0.tgz',
      registry: 'https://npmjs.com/package/test',
      version: 'https://npmjs.com/package/test/v/1.0'
  }
}
if (data[license]) expected.license = { declared: data[license] }
assert.deepEqual(result, expected)
```

Since the `urls` part is reused so much thoughout the tests, perhaps that should be a constant somewhere as well.